.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Type of Change
+
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+### Related issues
+
+> Fix [#1]()
+
+### 📋 Checklist
+
+<!--
+  Please keep your PR as a draft or add the Work In Progress label until it is ready to be reviewed.
+  Please review this checklist yourself before marking your PR as ready for review.
+-->
+
+When making changes, please review the following:
+
+#### Development
+
+- [ ] The code changed/added as part of this pull request has been covered with tests
+- [ ] All tests related to the changed code pass in development
+
+#### Code review
+
+- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
+- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
```

Comment:
I don't think we need a new label since the PR already has Draft/Ready states as you mentioned on line 14.

Reply:
```suggestion
```